foladipo/docs-cabinet-cp2
# Feature/148543657/add pagination

#### What does this PR do?

This PR enables a user to page through content using infinite scrolling.

#### Description of task to be completed

Enable a user to page through content.

#### How should this be manually tested?

- Follow the instructions stated in this project's README to set up your local instance of this app.

- Consequently, you should have signed up for a new account.

- Upon logging in, click the menu button and create 15 or more documents.

- Log out.

- Log in again and you'll see a list of documents that you can page through via infinite scrolling.

- This is available on the 'My documents' page too, and on the 'Users' page too (IF you're logged in as an admin).

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

148543657

#### Screenshots

N/A
